docs/topics/mta-web-creating-applications.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-applications_{context}"]
+= Creating an application profile
+
+You can create an application profile in order to add an application to the *Application Invcentory* for assessment and analysis.
```

Comment:
spelling : Inventory

Reply:
fixed

Reply:
@sshveta  I fixed this -- please review the PR again.